docs/source/custom-graphs.rst

```diff
@@ -56,6 +56,30 @@ analyze pipeline:
    get(dsk, 'store')  # executes in parallel
 
 
+
+Keyword arguments in custom Dask graphs
+---------------------------------------
+
+Sometimes, you may want to pass keyword arguments to a function 
+in a custom Dask graph.
```

Comment:
Maybe add another sentence mentioning `dask.utils.apply`, and (give it a
```
:func:`dask.utils.apply`
```
link) if that can get added to the list at https://github.com/dask/dask/blob/main/docs/source/api.rst#utilities

Reply:
Nice suggestion, done!